examples/conf.py

```diff
@@ -112,35 +112,9 @@ def setup(app: Sphinx):
 # a list of builtin themes.
 
 # theme options
-html_theme = "pydata_sphinx_theme"
+html_theme = "pymc_sphinx_theme"
 html_theme_options = {
-    "icon_links": [
```

Comment:
Why are you removing these?

Reply:
answered in https://github.com/pymc-devs/pymc/pull/6296/files#r1022185629